source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,188 @@
+ - Starting with the Calls version shipping with v7.7, there's now a minimum version requirement when using the external RTCD service. This means that if Calls is configured to use the external service, customers need to upgrade RTCD first to at least version 0.8.0 or the plugin will fail to start.
+ - Boards will be served as an in-built product from within Mattermost server instead of a plugin. If you want to disable this, please set the ``MM_FEATUREFLAGS_BoardsProduct`` env var to "false" (without the quotes). In this mode, the boards plugin will remain disabled.
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.5, please read the other [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
+
+### Compatibility
+ - Updated the minimum version of MacOS to 11+.
+
+### Highlights
+
+#### Calls
+ - [Audio calling and screen sharing](https://docs.mattermost.com/configure/calls-deployment.html) in channels is now generally available to all Mattermost customers.
+ - Updated [the keyboard shortcut](https://docs.mattermost.com/channels/keyboard-shortcuts-for-channels.html#calls-shortcuts) to start and join calls.
+
+#### Boards
+ - Boards now supports file attachments, including PDFs, images, videos, and any other file types.
+ - Users can now [drag and drop boards and categories](https://docs.mattermost.com/boards/navigate-boards.html#manage-boards-on-the-sidebar) on the sidebar and organize them in any order they prefer.
+ - The [template picker](https://docs.mattermost.com/boards/work-with-boards.html#choose-a-board-template) has been improved to make it easier for users to find the best template for their project.
+ 
+#### Playbooks
+ - Added an option to run playbooks without creating a new channel every time in order to reduce the unnecessary overhead.
+ - In addition to the daily digest, users can now also view [a task inbox](https://docs.mattermost.com/playbooks/work-with-tasks.html#task-inbox) from the global header bar while in Playbooks.
+
+#### Message Priority and Acknowledgments
+ - Added [message priority labels](https://docs.mattermost.com/channels/message-priority.html) to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts (Professional).
+
+#### Global Drafts
+ - Added [a centralized Drafts view](https://docs.mattermost.com/channels/send-messages.html#draft-messages) for draft messages.
+ 
+#### ServiceNow Integration
+ - ServiceNow customers can now access and share their ServiceNow data from within Mattermost.
+ 
+#### ServiceNow Virtual Agent Integration
+ - The [Mattermost ServiceNow Virtual Agent Integration](https://www.loom.com/share/36f60299c6c349729160d6d8bce75c3c) makes it easy for employees and customers to resolve issues fast.
+
+### Improvements
+
+#### User Interface (UI)
+ - Self-hosted admins can now purchase licenses in-app when service setting ``SelfHostedPurchase`` is true.
```

Comment:
@marianunez is this in line with our setup of "hiding" this feature until security gives us the green light?

Reply:
The feature is still available but it will show a message to connect to customers.mattermost.com until we enable it in the portal so this change still applies as the portal will be enabled shortly after release.

Reply:
@johndavidlugtu @marianunez In that case I'd remove it from the changelog (if the security review hasn't happened by Monday) and then add it back to the changelog once it has been security approved. Does this work?

Reply:
That works @amyblais , thanks!

Reply:
+1 thanks @amyblais!